src/core_ocean/shared/mpas_ocn_tendency.F

```diff
@@ -326,6 +330,17 @@ subroutine ocn_tend_vel(tendPool, statePool, forcingPool, diagnosticsPool, meshP
       !
       call ocn_vel_vadv_tend(meshPool, normalVelocity, layerThicknessEdge, vertAleTransportTop, tend_normalVelocity, err)
 
+      ! Compute tidal potential forcing, if needed 
+      if (config_use_tidal_potential_forcing) then
+        call ocn_compute_tidal_potential_forcing(meshPool, forcingPool, diagnosticsPool, err)
+      end if
+
+      if (config_use_tidal_potential_forcing .and. config_time_integrator == 'RK4') then
+        call mpas_pool_get_array(forcingPool, 'tidalPotentialZMid', zMid)
+      else
+        call mpas_pool_get_array(diagnosticsPool, 'zMid', zMid)
+      end if
```

Comment:
@pwolfram and @sbrus89, okay, so his is the modification to the code that actually changes the physics?  That wasn't clear to me at all.

Reply:
Now that I understand, it seems like a clever things to do.  I'll give @pwolfram a call soon to make sure I understand fully.

Reply:
I tried to make things more explicit in my recent changes.